sensing/preprocessor/pointcloud/livox/livox_tag_filter/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.5)
+project(livox_tag_filter)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wno-unused-parameter -Wall -Wextra -Wpedantic)
+endif()
+
+### Dependencies
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+find_package(PCL REQUIRED)
+include_directories(${PCL_INCLUDE_DIRS})
+
+# Target
+## livox_tag_filter_node
+ament_auto_add_executable(livox_tag_filter_node
+  src/livox_tag_filter_node/main.cpp
+  src/livox_tag_filter_node/livox_tag_filter_node.cpp
```

Comment:
I think it's better to register this node as a component. How do you think?

Reply:
made it to component in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/368/commits/cd58dc47a1ba24312bde9cf0712a614e7dc88bbd